.gitignore

```diff
@@ -1,2 +1,3 @@
 # Project folders.
 temp/
+tmp/
```

Comment:
Should we harmonize things to use a single folder for temporary stuff?

Reply:
I'm fine changing this repo's temp folder name to `tmp` since this is what build scripts use.